src/components/Navbar.js

```diff
@@ -1,58 +1,71 @@
 import React from 'react';
```

Comment:
@phi-h-nguyen  check out this file for combining the two selected line index states into one (only use the parent)